SR2021/2021-01-12-team-numbers.md

```diff
@@ -0,0 +1,16 @@
+---
+to: SR2021 institutions with inactive teams
+subject: Student Robotics Teams
+---
+
+Hi,
+
+If you're reading this email, it means you have at least 1 team from your institution who seem inactive on Discord, or who have few members.
```

Comment:
I wonder if a slightly friendlier tone might help here? I just re-read this and it sounds very direct.

Perhaps something like this is a bit kinder?
```suggestion
We noticed that one or more of your teams doesn't seem active on Discord, or has very few members. We want to ensure that every team has the opportunity to shine and gets the support they need.
```

Reply:
100% agree! :tada: